lte/gateway/deploy/agw_post_install.sh

```diff
@@ -79,3 +83,35 @@ if [ -n "$INFO" ]; then
     echo "INFO:"
     printf "%s" "$INFO"
 fi
+
+echo -e "${WHITE}Checking for Root Certificate"
+CA=/var/opt/magma/tmp/certs/rootCA.pem
+if [ -d "/var/opt/magma/tmp/certs/" ]; then
+    if [ -f "$CA" ]; then
+        echo -e "${GREEN}$CA exists"
+    else
+    echo -e "${RED}Check Root CA in /var/opt/magma/tmp/certs/"
+    echo -e "${RED}Access Gateway configurations failed"
+    fi
+fi
+
+echo -e "${WHITE}Checking for Control Proxy"
+CP=/var/opt/magma/configs/control_proxy.yml
+if [ -d "/var/opt/magma/configs/" ]; then
+    if [ -f "$CP" ]; then
+        echo -e "${GREEN}$CP exists"
+    else
+    echo -e "${RED}Check Control Proxy Configs in /var/opt/magma/configs/"
```

Comment:
Can we additionally check if we have fluend/bootstrap_address/cloud_address is present in the control_proxy file

Reply:
Also can we provide more information on missing file or missing attribute